core/types/block.go

```diff
@@ -407,6 +416,7 @@ func (h *Header) RPCMarshalHeader() map[string]interface{} {
 		"etxEligibleSlices": h.EtxEligibleSlices(),
 		"stateLimit":        hexutil.Uint64(h.StateLimit()),
 		"stateUsed":         hexutil.Uint64(h.StateUsed()),
+		"secondaryCoinbase": h.SecondaryCoinbase(),
```

Comment:
Can you do h.SecondaryCoinbase().MixedCaseAddress() here

Reply:
done

Reply:
Sorry, donge again